extensions/default/src/DicomWebDataSource/wado/retrieveMetadataLoader.js

```diff
@@ -12,20 +12,27 @@ export default class RetrieveMetadataLoader {
    * @param {Array} studyInstanceUID Study instance ui to be retrieved
    * @param {Object} [filters] - Object containing filters to be applied on retrieve metadata process
    * @param {string} [filter.seriesInstanceUID] - series instance uid to filter results against
-   * @param {Function} [sortSeries] - Custom sort function for series
+   * @param {Object} [sortCriteria] - Custom sort criteria used for series
+   * @param {Function} [sortFunction] - Custom sort function for series
+   * @param {number} [thresholdLargeStudy] - Threshold used for classify a study as a large studies
+   * @param {number} [firstGroupSize] - In case of a large study, fetch first <firstGroupSize> series to display the first image fast to the user
    */
   constructor(
     client,
     studyInstanceUID,
     filters = {},
-    sortCriteria,
-    sortFunction
+    sortCriteria = undefined,
+    sortFunction = undefined,
+    thresholdLargeStudy = 100,
```

Comment:
I recommend more meaningul names 

such as `largeStudySeriesCountThreshold` and `initialSeriesFetchSize`